packages/lib-classifier/src/components/Classifier/components/SubjectViewer/SubjectViewer.js

```diff
@@ -0,0 +1,40 @@
+import { inject, observer } from 'mobx-react'
+import PropTypes from 'prop-types'
+import React from 'react'
+import getViewerComponent from './helpers/getViewerComponent'
+
+function storeMapper (stores) {
+  return {
+    currentSubject: stores.classifierStore.subjects.current
+  }
+}
+
+@inject(storeMapper)
+@observer
+class SubjectViewer extends React.Component {
+  getViewerComponents() {
+    return this.props.currentSubject.locations.map(getViewerComponent)
```

Comment:
Is this just for initial demo purposes? It's probably not going to work for real subjects. At least, this is one of the more brittle bits of the existing subject viewer design so maybe not something we should copy across to the new classifier.

Reply:
The way I imagined it working was this producing an array of viewer components, and having another function plug those viewers into a layout. It works for now as it's a single location, but won't work with multiple locations

Reply:
> this producing an array of viewer components, and having another function plug those viewers into a layout

That's the design that the current classifier uses, with the FileViewer component. It assumes we have distinct viewers for each individual subject location, which turns out not to be true in practice.